lib/teiserver/lobby/libs/lobby_restrictions.ex

```diff
@@ -291,37 +293,16 @@ defmodule Teiserver.Lobby.LobbyRestrictions do
   """
   @spec is_noob_title?(String.t()) :: boolean()
   def is_noob_title?(title) do
-    title =
-      title
-      |> String.downcase()
+    anti_noob_regex = ~r/no (noob|newb|nub)/i
+    noob_regex = ~r/\b(noob|newb|nub(s|\b))/i
```

Comment:
what does `\b` do?

Reply:
It's a word boundary. So nub or nubs would be a match. But a player named "Anubis" who has their name in the room title would not be a match.